.eslintrc.json

```diff
@@ -14,6 +14,14 @@
     "no-var": 2,
     "no-console": 2,
     "prefer-const": 2,
-    "object-curly-spacing": [2, "always"]
+    "object-curly-spacing": [2, "always"],
```

Comment:
Let's move any new rules to a different PRs since I wouldn't want to block just because of these (I don't agree with some of them).

Reply:
Sure, that's fine.